include/ipfs/client.h

```diff
@@ -377,6 +377,48 @@ class Client {
       /** [in] Key name (local, user-friendly name for the key). */
       const std::string& key_name);
 
+  /** Publish an IPNS name attached to a given value.
+   *
+   * Implements
+   * https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/NAME.md#name.publish.
```

Comment:
Should be https://github.com/ipfs/interface-js-ipfs-core/blob/master/SPEC/NAME.md#namepublish

Reply:
fixed, will push shortly